lib/models/ib/ib_showcase.dart

```diff
@@ -0,0 +1,47 @@
+import 'dart:convert';
+
+import 'package:flutter/material.dart';
+
+class IBShowCase {
+  bool nextButton, prevButton, contentButton, drawerButton;
+
+  IBShowCase({
+    @required this.nextButton,
+    @required this.prevButton,
+    @required this.contentButton,
```

Comment:
since this variable is for table of contents, we can enhance it and name it as `tocButton`

Reply:
okk. :+1: